packages/dd-trace/src/tracer.js

```diff
@@ -62,15 +62,16 @@ class DatadogTracer extends Tracer {
     const tracer = this
 
     return function () {
+      let theseOptions = options
```

Comment:
The name doesn't really describe what this is doing. Maybe something like `localOptions`?

Reply:
I can't remember if this change is even necessary. If it is, I'll change the name.

Reply:
It's not. I'll back this change out.

Reply:
Is it not? The function changes `options` every time it's called. While this could be safe if it's done synchronously, it's still not ideal.